django_ipam/base/models.py

```diff
@@ -76,3 +79,29 @@ def clean(self):
         for ip in swapper.load_model("django_ipam", "IpAddress").objects.filter().values():
             if self.id != ip["id"] and ip_address(self.ip_address) == ip_address(ip["ip_address"]):
                 raise ValidationError({'ip_address': _('IP address already used.')})
+
+    def export_csv(self, subnet_id, writer, queryset):
+        IpAddress = swapper.load_model("django_ipam", "IpAddress")
+        subnet = swapper.load_model("django_ipam", "Subnet").objects.get(pk=subnet_id)
+        writer.writerow([subnet.name, ])
+        writer.writerow([subnet.subnet, ])
+        writer.writerow('')
+        fields = [IpAddress._meta.get_field('ip_address'), IpAddress._meta.get_field('description')]
+        writer.writerow(field.name for field in fields)
+        for obj in queryset:
+            row = []
+            for field in fields:
+                row.append(str(getattr(obj, field.name)))
+            writer.writerow(row)
+
+    def import_csv(self, file):
+        IpAddress = swapper.load_model("django_ipam", "IpAddress")
+        Subnet = swapper.load_model("django_ipam", "Subnet")
+        reader = csv.reader(StringIO(file.read().decode('utf-8')), delimiter=',')
+        subnet = Subnet.objects.get_or_create(name=next(reader)[0].strip(), subnet=next(reader)[0].strip())[0]
```

Comment:
this is unreadable, please break it on multiple line.
Please don't call `get_or_create` because it doesn't perform validation when creating objects.
You have to call `full_clean` and abort the operation gracefully if the validation fails.